metrilo_open_api_specification.yml

```diff
@@ -318,6 +372,48 @@ components:
                   example:
                     - early_subscriber
                     - big_spender
+    customerTagObject:
+      allOf:
+        - $ref: '#/components/schemas/visitorIdentityObject'
+        - type: object
+          required:
+            - createdAt
+            - email
+            - tags
+          properties:
+            email:
+              type: string
+              description: The email of the customer.
+              example: johnybravo@metrilo.com
+            tags:
+              type: array
+              items:
+                  type: string
+                  description: Tag labels that are assigned to user in the Metrilo. (tags are merged, not overwritten)
```

Comment:
`Tag labels that are assigned to customer in the Metrilo. (tags are merged, not overwritten)`